packages/samples/headless-ssr-commerce/components/product-list.tsx

```diff
@@ -38,6 +38,9 @@ export default function ProductList() {
           <button onClick={() => addToCart(cartMethods!, product)}>
             Add to cart
           </button>
+          <button onClick={() => addToCart(cartMethods!, product)}>
```

Comment:
Why are there three add to cart buttons?

Reply:
results of bad merges oops ! let me fix it there should only be one in master